lib/ui/home/widgets/activities/activity_widget.dart

```diff
@@ -120,14 +120,14 @@ class ActivityWidget extends StatelessWidget {
     return null;
   }
 
-  String _titleForActivityType(ActivityType activityType) {
+  String _titleForActivityType(BuildContext context, ActivityType activityType) {
     switch (activityType) {
       case ActivityType.walking:
-        return Strings.activityWalking;
+        return S.of(context).activityWalking;
       case ActivityType.running:
-        return Strings.activityRunning;
+        return S.of(context).activityRunning;
       case ActivityType.biking:
-        return Strings.activityBiking;
+        return S.of(context).activityBiking;
```

Comment:
przydałoby się sprawdzić w jaki sposób działa `S.current` i jeżeli nie różni się niczym od użycia contextu to tego typu mapowania można by było wydzielić do jakiegoś notifiera/w inne miejsce poza sam UI

Reply:
Zbadam temat. Skoro w easy_localization mieli problem z tym, żeby nasłuchiwać na zmiany locale systemowego to pewnie jest to trywialny mechanizm, no chyba że coś się zmieniło 🤔

Reply:
Dokumentacja do intl niewiele mówi.
W praktyce `current` wydaje się działać tak samo jak `.ofContext()` jeśli chcemy używać systemowego locale. Po prostu przy zmianie local widgety się przebudowują i wtedy `current` też się odświeża.

Jedyny case jaki znalazłem, gdzie jest różnica to overridowanie locale: https://docs.google.com/document/d/10e0saTfAv32OZLRmONy866vnaw0I2jwL8zukykpgWBc/edit#heading=h.d0lemmo1qom
Jeśli byłby jakiś chooser wewnątrz apki to mogło by się chyba przydać :thinking: 

Podsumowując zostawiłbym tak jak jest, skoro context już jest dostępny praktycznie wszędzie i obecnie tak jest w kodzie.